geode-core/src/integrationTest/java/org/apache/geode/internal/jta/functional/CacheJUnitTest.java

```diff
@@ -440,7 +444,7 @@ public void testScenario5() throws Exception {
       // now current region should point to region1, as done from within
       // getRegionFromCache method...
       String current_fullpath = jtaObj.currRegion.getFullPath();
-      assertEquals("failed retirieving current fullpath", "/" + DEFAULT_RGN + "/region1",
+      assertEquals("failed retirieving current fullpath", SEPARATOR + DEFAULT_RGN + "/region1",
```

Comment:
Did you miss this one?, shouldn't it be `SEPARATOR + DEFAULT_RGN + SEPARATOR + "region1"`?